costs/model-costs.jsonl

```diff
@@ -1,16 +1,86 @@
 {
     "model": "models/gemini-1.5-pro",
     "model_family": "gemini",
-    // "model_type": "text", 
+    "request_context_window": "Over 128k",
+    "model_type": "text", 
     "max_tokens_per_minute": 4000000, 
     "max_requests_per_minute": 1000, 
-    // "request_context_window": 4096,
-    // "max_output_tokens": 2048, 
-    // "input_cost_10k": 0.035, 
-    // "output_cost_10k": 0.105,
+    "max_output_tokens": null, 
+    "input_cost_10k": 0.025, 
+    "output_cost_10k": 0.10,
     "developer": "Google DeepMind", 
     "provider": "Google Cloud Platform", 
-    "pricing_page": "https://ai.google.dev/pricing#1_5flash", 
-    // "pricing_plan": "Monthly", 
-    // "updated": "2024-10-24"
+    "pricing_page": "https://ai.google.dev/pricing#1_5pro", 
+    "pricing_plan": "Pay-as-you-go", 
+    "updated": "2024-10-25"
+}
+
+{
+    "model": "models/llama-3.1-405b",
+    "model_family": "llama", 
+    "model_type": "text", 
+    "max_tokens_per_minute": 780, 
+    // "max_requests_per_minute": 50, 
+    "request_context_window": "128k",
+    // "max_output_tokens": 131072, 
+    "input_cost_10k": 0.0533, 
+    "output_cost_10k": 0.16, 
+    "developer": "Meta AI", 
+    "provider": "Azure", 
+    "pricing_page": "https://azuremarketplace.microsoft.com/en-us/marketplace/apps/metagenai.meta-llama-3-1-405b-instruct-offer?tab=PlansAndPrice", 
+    "pricing_plan": "Pay-as-you-go", 
+    "updated": "2024-10-25"
+}
+
+{
+    "model": "models/gpt-4o", 
```

Comment:
This should be "gpt-4o" based on https://platform.openai.com/docs/models/. Same applies to the other models too.